lxd/device/nic_bridged.go

```diff
@@ -1188,3 +1194,57 @@ func (d *nicBridged) networkDHCPv6CreateIAAddress(IP net.IP) []byte {
 	binary.BigEndian.PutUint32(data[24:28], uint32(0))                    // Valid lifetime
 	return data
 }
+
+// setupBridgePortVLANs configures the bridge port with the specified VLAN settings in device config.
+func (d *nicBridged) setupBridgePortVLANs(hostName string) error {
+	// Enable vlan_filtering on bridge if needed.
```

Comment:
Comment is inaccurate, we're just checking

Reply:
Will fix in my OVS branch.